victims/__init__.py

```diff
@@ -47,3 +47,10 @@ def force_https(request):
 @jinja.template("about.html")
 async def about(request):
     return {"title": TITLE, "url_path": "/about.html"}
+
+
+@app.route("/data.json")
+async def data(request):
+    cases = await get_cases()
+    cases = [case.to_JSON() for case in cases]
```

Comment:
There's no `case.to_JSON()` anymore, right?

Reply:
you're right, my bad